cli/lsp/diagnostics.rs

```diff
@@ -229,11 +229,12 @@ fn ts_json_to_diagnostics(
     .collect()
 }
 
+// None is returned if getting diagnostics was cancelled.
```

Comment:
Maybe turn this into a proper docblock comment?  Otherwise it feels just like a stray comment.